server/controllers/TripController.js

```diff
@@ -0,0 +1,44 @@
+const TripModel = require('../models/TripModel')
+const uuid = require('uuid')
+
+class TripController {
+  constructor() {}
+
+  async getAll() {
+    try {
+      const trips = await TripModel.find()
+      return trips
+    } catch (error) {
+      console.error('Error while retrieving trips:', error)
+      throw error
+    }
+  }
+
+  async createTrip(tripData) {
+    try {
+      const newTrip = await TripModel.create({
+        // eslint-disable-next-line node/no-unsupported-features/es-syntax
+        ...tripData,
+        _id: uuid.v4(),
```

Comment:
do we need to specify the ID? would mongo do that for us?

Reply:
we can use either, i think the mongo id is a bit different from uuid. if im not mistake